swift-sdk/Internal/InAppManager.swift

```diff
@@ -134,7 +136,13 @@ class InAppManager: NSObject, IterableInAppManagerProtocolInternal {
         }
     }
     
-    func remove(message: IterableInAppMessage, location: InAppLocation, source: InAppDeleteSource) {
+    func remove(message: IterableInAppMessage) {
```

Comment:
Is this necessary if `func remove(message: IterableInAppMessage, location: InAppLocation = .unknown, source: InAppDeleteSource = .unknown)` exists as a function with defaults?

Reply:
Yes. Because that was part of public API. It will work fine with Swift but will be problematic for Objective C because it doesn't have default parameters.